<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageExportOptionsDialog</class>
 <widget class="QDialog" name="ImageExportOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Export Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="margin">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="channelsGroupBox">
     <property name="title">
      <string>Channels</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="blueChannelCheckBox">
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="redChannelCheckBox">
        <property name="text">
         <string>Red</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="greenChannelCheckBox">
        <property name="text">
         <string>Green</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="numberOfCopiesGroupBox">
     <property name="title">
      <string>Number of Copies</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="numberLabel">
        <property name="text">
         <string>Number:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="numberEdit">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageExportOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageExportOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
